util/get-toolchain.py

```diff
@@ -37,7 +38,10 @@
 
 def get_available_toolchain_info(version, kind):
     assert kind in ASSET_PREFIXES
-
+    
+    # Check if we're on x86_64
+    arch = "x86_64" if platform.machine() == 'x86_64' else "aarch64"
```

Comment:
The python docs say `platform.machine()` could return `AMD64`, could you add that to the condition as well please?

https://docs.python.org/3/library/platform.html#platform.machine

Reply:
From what I can find this returns platform-speciric string and on Linux it's always going to use the arch inside uname.

That said, we probably should have logic here to only accept known "machine"s rather than saying anything except x86 is aarch64.